gulpfile.babel.js

```diff
@@ -364,26 +377,30 @@ gulp.task('components', ['demos'], () => {
       });
     })())
     .pipe(applyTemplate())
-    .pipe($.rename(path => path.basename = 'index'))
+    .pipe($.rename(path => {
+      path.basename = 'index';
+    }))
```

Comment:
It doesn't :) But we have a linter rule that doesn't allow assignments in return statements. One option was to disable the rule, the other option was to make this not be a return statement. I chose the latter :)